packages/card-react/src/TaskCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react";
+import cn from "classnames";
+import { PaddingOptions } from "./types";
+
+export interface TaskCardProps extends PaddingOptions {
+    /**
+     * Bakgrunnsfargen til kortet
+     * @default "hvit"
+     */
+    bgColor?: "hvit" | "snohvit" | "sand" | "dis";
+    /**
+     * Skal kortet ha skygge?
+     * @default false
+     */
+    shadow?: boolean;
+    className?: string;
+}
+
+export const TaskCard: FC<TaskCardProps> = ({
+    bgColor = "hvit",
+    shadow = false,
+    padding = "l",
+    topPadding,
+    children,
+    className,
```

Comment:
Jeg er fan av å samle `...rest` og spre dem på rotelementet. Gjør det enklere for folk å legge til HTML-attributter.

Reply:
Ikke så mye å spreade her foreløpig, men kan godt legge til en `...rest` preemptivt 😊